triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/AbstractFuncTests.scala

```diff
@@ -464,15 +463,12 @@ abstract class AbstractFuncTests
             QualifiedName.assertFromString("TemplateIdFilter:testTwo"),
             party,
           )
+          _ <- create(client, party, one(party))
+          _ <- create(client, party, two(party))
           (acs, offset) <- runner.queryACS()
-          // 2 for the creates from the test
-          // 2 for the completions from the test
           // 1 for the create in the trigger
           // 1 for the completion from the trigger
```

Comment:
```suggestion
          // We take 2 messages: 
          // - 1 for the create in the trigger
          // - 1 for the completion from the trigger
```

Reply:
I think it makes more sense to address this comprehensively for all of them rather than just changing this so I’ll leave it out of this PR but will change them once I find the time.

Reply:
:+1: